documentation/src/main/asciidoc/user_guide/persistence_leveldb.adoc

```diff
@@ -41,12 +39,12 @@ Configuration cacheConfig = new ConfigurationBuilder().persistence()
 |===============
 |Parameter|Description
 |location|Directory to use for LevelDB to store primary cache store data.  Directory will be auto-created if it does not exit.
-|expiredLocation| Directory to use for LevelDB to store expiring data pending to be purged permanently.  Directory will be auto-created if it does not exit. 
+|expiredLocation| Directory to use for LevelDB to store expiring data pending to be purged permanently.  Directory will be auto-created if it does not exit.
 | expiryQueueSize |Size of the in-memory queue to hold expiring entries before it gets flushed into expired LevelDB store
-|clearThreshold| There are two methods to clear all entries in LevelDB.  One method is to iterate through all entries and remove each entry individually.  The other method is to delete the database and re-init.  For smaller databases, deleting individual entries is faster than the latter method.  This configuration sets the max number of entries allowed before using the latter method 
+|clearThreshold| There are two methods to clear all entries in LevelDB.  One method is to iterate through all entries and remove each entry individually.  The other method is to delete the database and re-init.  For smaller databases, deleting individual entries is faster than the latter method.  This configuration sets the max number of entries allowed before using the latter method
 | compressionType |Configuration for LevelDB for data compression, see CompressionType enum for options
-| blockSize | Configuration for LevelDB - see link:$$http://leveldb.googlecode.com/svn/trunk/doc/index.html$$[documentation] for performance tuning 
-| cacheSize | Configuration for LevelDB - see link:$$http://leveldb.googlecode.com/svn/trunk/doc/index.html$$[documentation] for performance tuning 
+| blockSize | Configuration for LevelDB - see link:$$https://github.com/Level/levelup/blob/master/README.md$$[documentation] for performance tuning
+| cacheSize | Configuration for LevelDB - see link:$$https://github.com/Level/levelup/blob/master/README.md$$[documentation] for performance tuning
```

Comment:
The proper link is `https://github.com/google/leveldb/blob/master/README.md` - `levelup` is just a NodeJS wrapper.